### DISCUSSION TITLE
Supabase Google Login is not working, is that my fault? I couldn't solve that.

### DISCUSSION THREAD

**Discussion by okandavut** [supabase/supabase#15885] [Questions]
Hi,

I have google login in my React (vite) application. There is no special configuration and it was working before. But now I after redirect from google to application I have access token and other fields in url but session is not setting to browser storage.

version of package :     "@supabase/supabase-js": "^2.26.0",

using this method : signInWithOAuth

<img width="637" alt="image" src="https://github.com/supabase/supabase/assets/10600157/c781e881-b599-47b8-a86c-dd3b0418ebc1">

I also saw this in Supabase Auth logs ;: GOTRUE_JWT_ADMIN_GROUP_NAME not supported by Supabase's GoTrue, will be removed soon

On App.jsx I am checking like this: 

<img width="614" alt="image" src="https://github.com/supabase/supabase/assets/10600157/76680b34-0885-4ff9-9367-a2b0fe4e0497">

**Comment by silentworks** [comment]
I have created an example project using just React with react-router and I'm not able to replicate the issue you are having. Maybe take a look at the repo and try running the example to see if there are any differences between the example code and your code. https://github.com/supabase-community/supabase-by-example/tree/main/oauth-flow/react

**Reply by okandavut** [reply]
Thank you for helping me! actually I tried also same code I shared with Next.js project and it works fine I really don't understand what is the issue. Because it was working.
